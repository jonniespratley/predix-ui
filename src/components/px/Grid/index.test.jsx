import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import PxGrid from './';

describe('Grid', () => {
  test('should render', () =>{
    const tree = renderer.create(<PxGrid/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
