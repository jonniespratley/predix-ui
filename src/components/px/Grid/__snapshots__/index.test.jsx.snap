// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render 1`] = `
.c0 {
  display: grid;
}

<div
  className="c0"
/>
`;
